src/midi/midi.cpp

```diff
@@ -80,15 +80,10 @@ MidiHandler Midi_none;
 
 #if defined(MACOSX)
 
-#if defined(C_SUPPORTS_COREMIDI)
 #include "midi_coremidi.h"
-#endif
-
-#if defined(C_SUPPORTS_COREAUDIO)
 #include "midi_coreaudio.h"
-#endif
 
```

Comment:
Should we keep the guards here too?
I think we build with gcc, one of the headers fails to import properly because it has objC content that only  clang can parse.

Reply:
I moved defines inside header files, I think there's no reason to move them back outside again…

Both CoreAudio and CoreMIDI are not compilable using GCC because Apple's "frameworks" work only with Apple compilers - features are disabled by both autoconf and meson in this case.

Reply:
Makes sense that further guards are in located inside rhese two midi_(macAPI).h files, instead of polluting this generic source.